sdk/appcenter-storage/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     api project(':sdk:appcenter')
 
     implementation 'com.google.code.gson:gson:2.8.5'
+    compile project(path: ':sdk:appcenter-identity')
```

Comment:
Remove that, we must not depend on Identity directly, everything you need is on the core.

Reply:
You're using the wrong class actually.